### ISSUE TITLE
[Bug] 当前的xattetion 在pto后端在copy_l1_to_l0b 切片场景下出现越界问题

### ISSUE THREAD

**Issue by liuwenda4** [tile-ai/tilelang-ascend#1293]
## Bug Description
```
T.copy(
    shared_v_stack_l1[chunk, :, :],   # 源：[4, 128, 128] 的一个切片
    mma_l0b[mma_slot, :, :],           # 目的：[128, 128]
)
```
对应生成的
```
TileMatL1<bfloat16_t, 256, 128, 256, 128> shared_v_stack_l1_temp_0;  // 源 tile: 256 行
TileMatL0B<bfloat16_t, 128, 128, 128, 128> mma_l0b_temp_8;            // 目的 L0B: 128 行
copy_l1_to_l0b<bfloat16_t, 128, 128, 256, 128>(...);                  // tile_row=256 > 128 → 崩
```
根因：
shared_v_stack_l1[chunk, :, :] 是一个 3D buffer 的 slice，GetSliceInfo 用物理行数 512 去算 tile_row，算出 256 超过了 L0B 的 128 行容量。

在后端的问题：
src/target/codegen_ascend_pto.cc:1348-1361 — 取形状 + 算 tile_row：
```
ShapeInfo src_shape_info = GetSliceInfo(src_info.access_ptr);
ShapeInfo dst_shape_info = GetSliceInfo(dst_info.access_ptr);

int32_t tile_col = src_shape_info.col;
int32_t tile_row =
    is_a ? dst_shape_info.slice_row
         : FindBestTileRowB(src_shape_info.row, dst_shape_info.slice_row);
                              ^^^^^^^^^^^^^^^^^^^^
                              问题：用 row=512，不是 slice_valid_row=256
int32_t num_tiles = src_shape_info.row / tile_row;
                    ^^^^^^^^^^^^^^^^^^
                    同样用 row=512
if (num_tiles < 1)
  num_tiles = 1;
FindBestTileRowB（行 1325-1338）：
static int32_t FindBestTileRowB(int32_t src_row, int32_t min_row) {
  int32_t best = min_row;
  for (int32_t d = 2; d * d <= src_row; ++d) {
    if (src_row % d == 0) {
      int32_t cand1 = src_row / d;
      int32_t cand2 = d;
      if (cand1 < src_row && cand1 >= min_row && cand1 > best) best = cand1;
      if (cand2 < src_row && cand2 >= min_row && cand2 > best) best = cand2;
    }
  }
  return best;
}
输入 (512, 128)：
- d=2 → cand1=256, cand2=2 → 256≥128 且 >128 → best=256
- 其他因子都不比 256 大
- 返回 256
输入 (256, 128)（A 改后）：
- d=2 → cand1=128, cand2=2 → 128≥128 但不 >128 → best 不变
- 返回 128
GetSliceInfo（行 308-356） — row 和 slice_valid_row 的来源：
// row/col：buffer 声明的物理形状
row = shape[0].as<IntImmNode>()->value;   // 512  ← shared_v_stack_l1=[4,128,128] 展平
col = shape[1].as<IntImmNode>()->value;   // 128

// extent：本次 access_ptr 访问的有效元素数
extent = static_cast<int32_t>(extent_imm->value);  // 32768（2 个 chunk）
```

// slice_valid_row：本次实际有效行数
slice_valid_row = (extent / col) > 1 ? (extent / col) : 1;  // 256

is_slice = (extent != row * col);  // 32768 != 65536 → true
解决办法：
https://github.com/tile-ai/tilelang-ascend/pull/1283（PR 行 1354-1359）就是加一行 src_row，三处全换：
```
int32_t src_row = src_shape_info.is_slice ? src_shape_info.slice_valid_row
                                          : src_shape_info.row;
int32_t tile_row =
    is_a ? dst_shape_info.slice_row
         : FindBestTileRowB(src_row, dst_shape_info.slice_row);
int32_t num_tiles = src_row / tile_row;
```
把 FindBestTileRowB 的输入从物理 512 改成有效 256 → tile_row 从 256 变 128 → 匹配 L0B 容量。
## Environment

**Hardware:**
- NPU Device: [e.g. Ascend A2, A3]
- Other hardware info: [e.g. CPU, memory]
A3
**Software:**
- OS: [e.g. Ubuntu 20.04, EulerOS]
- CANN version: [e.g. 8.3.RC1]
- torch-npu version: [e.g. 2.6.0.RC1]
- Python version: [e.g. 3.10]
- TileLang-Ascend version: [e.g. commit hash or tag]
- Installation method: [wheel package / build from source]

## Code to Reproduce

```python
# Minimal reproducible code
import tilelang
import torch

# Your code here
```

## Error Message

```
Paste the full error message here
The error from device(chipId:0, dieId:0), serial number is 99, there is an exception of fftsplus aicore error, core id is 1, error code = 0, dump info: pc start: 0x124000024000, current: 0x1240000246cc, vec error info: 0, mte error info: 0x90000000b2, ifu error info: 0x375dbfa705800, ccu error info: 0x80400000108000a6, cube error info: 0, biu error info: 0, aic error mask: 0x6500020bd00028c, para base: 0x12c100000080.[FUNC:PrintCoreInfo][FILE:device_error_core_proc.cc][LINE:657]
       The extend info: errcode:(0, 0x8000000, 0) errorStr: The value of K_M_START_POS is illegal. fixp_error0 info: 0xb2, fixp_error1 info: 0x90, fsmId:0, tslot:2, thread:0, ctxid:0, blk:3, sublk:0, subErrType:4.[FUNC:PrintCoreInfo][FILE:device_error_core_proc.cc][LINE:670]
```

## Actual Behavior
A clear and concise description of what actually happened.

## Additional Context

**Programming Mode:**
- [ ] Developer mode (automatic sync/buffer management)
- [ ] Expert mode (manual T.Scope/T.barrier_all)
- [x] Mixed mode

**Backend:**
- [x] Ascend C & PTO backend
- [ ] AscendNPU IR backend

**Pass Configs (if applicable):**
```python
pass_configs = {
    tilelang.PassConfigKey.TL_ASCEND_AUTO_SYNC: ...,
    tilelang.PassConfigKey.TL_ASCEND_MEMORY_PLANNING: ...,
    # ...
}
```

**Operator Type:**
[e.g. GEMM, Flash Attention, Softmax, Convolution, etc.]
Flash Attention
**Additional Information:**
- Have you checked the [Programming Guide](../docs/TileLang-Ascend%20Programming%20Guide.md)?
- Have you searched existing issues?
- Any other context or screenshots?

**Comment by 2Sheldonlaw** [comment]
Fixed by #1314 + #1283.
- #1314（3b30ff04）：CopyL1ToL0Codegen 用 slice_valid_row 替代物理 row 作为 FindBestTileRowB 输入 → tile_row 256→128，匹配 L0B 容量（即本 issue 根因分析中的 src_row 方案）
- #1283（d2d34e95）：TROWEXPAND{SUB,MUL,DIV}_row_vec 改为模板类型推导 → 消除 <64,8> vs <1,8> 签名不匹配（即报错中的 6 个 template errors）

注：#1314（07-09 合入）早于 #1283（07-15），本 issue 基线 ce07b80c（07-03）两者皆缺。

已在最新分支按原始步骤验证：两个变体 target="pto" 编译通过，Kernel Output Match!。#1273 的精度问题亦已验证修复。可关闭。

**Comment by 2Sheldonlaw** [comment]
[https://github.com/tile-ai/tilelang-ascend/pull/1314](url)
这个合入的PR已经修复目前的issue，可以检查一下仓库最新的这个状态，应该可以close这个issue
